cmd/tesla-http-proxy/main.go

```diff
@@ -44,8 +44,14 @@ func main() {
 		host         string
 		port         int
 	)
-	config := cli.Config{Flags: cli.FlagPrivateKey}
-	var err error
+
+	config, err := cli.NewConfig(cli.FlagPrivateKey)
+
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to load credential configuration: %s\n", err)
+		return
+	}
```

Comment:
This will exit with status 0. We should move this below the deferred function and rely on that to print the error message and exit.